packages/shared-state/files/usr/libexec/rpcd/shared-state

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env lua                     
+--[[                                                                                 
+Shared State
+
+Copyright (c) 2023  Javier Jorge <jjorge@inti.gob.ar>
+Copyright (c) 2023  Instituto Nacional de Tecnología Industrial
+Copyright (C) 2023  Asociación Civil Altermundi <info@altermundi.net>
+
+This is free software, licensed under the GNU AFFERO GENERAL PUBLIC LICENSE Version 3
+]] --                                                                                 
+local ubus = require "ubus"
+local utils = require('lime.utils')
+local shared_state = require("shared-state")
+local json = require 'luci.jsonc'
+
+require("nixio.util")
+
+local function get_from_sharedState(msg)
+    local sharedState = shared_state.SharedState:new(msg.data_type, 
+		nixio.syslog)
+    local resultTable = sharedState:get()
+    utils.printJson(resultTable)
+end
+
+local function get_from_sharedState_muti_writer(msg)
```

Comment:
`sed -i s/muti_writer/multi_writer/g` ;-)
also better not mixing up underscore (python like) and camel case, either keep _ o camel case but not both ;)